xml/_filter.xml

```diff
@@ -153,9 +156,11 @@
       <typeFilter name="*" expose="false" />
     </namespaceFilter>
     <namespaceFilter name="Microsoft.Extensions.Logging.Console.Internal">
+      <typeFilter name="IConsole" expose="true" />
       <typeFilter name="*" expose="false" />
     </namespaceFilter>
     <namespaceFilter name="Microsoft.Extensions.Logging.EventLog.Internal">
+      <typeFilter name="IEventLog" expose="true" />
```

Comment:
This looks to be `internal`. And I don't see it in this namespace. I think it should be reverted.

Reply:
This one's the property type for [EventLogLogger.EventLog](https://docs.microsoft.com/en-us/dotnet/api/microsoft.extensions.logging.eventlog.eventloglogger.eventlog?view=dotnet-plat-ext-2.2#Microsoft_Extensions_Logging_EventLog_EventLogLogger_EventLog) and [EventLogSettings.EventLog](https://docs.microsoft.com/en-us/dotnet/api/microsoft.extensions.logging.eventlog.eventlogsettings.eventlog?view=dotnet-plat-ext-2.2#Microsoft_Extensions_Logging_EventLog_EventLogSettings_EventLog).